memcached-image/memcached-image.xsl

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+<xsl:output method="xml" encoding="UTF-8" indent="yes"/>
+<xsl:template match="/">
+<xsl:comment> OBS-ExclusiveArch: aarch64 x86_64 </xsl:comment>
+<xsl:comment>
+  This file is generated,
```

Comment:
this comment is confusing?

Reply:
Why is it confusing? It tells you in the generated .kiwi file is autogenerated and not the source. I find this very helpful because it tells me, that I should not create SRs against the kiwi files but against the real source

Reply:
yeah, the generated comment should be in the generated source not in the input (xsl). I guess its a macro logic that we would need to substitute this string with something that the source does not include but merely reference.